websocket-tcp-chat/src/main.rs

```diff
@@ -183,6 +188,34 @@ impl StreamHandler<ws::Message, ws::ProtocolError> for WsChatSession {
     }
 }
 
+impl WsChatSession {
+    /// helper method that sends ping to client every second.
+    ///
+    /// also this method check heartbeats from client
+    fn hb(&self, ctx: &mut ws::WebsocketContext<Self, WsChatSessionState>) {
+        ctx.run_later(Duration::new(1, 0), |act, ctx| {
```

Comment:
Consider `run_interval`?

Reply:
Yes. I answered in the previous pull request, sorry. I change it also in the implementation of session.rs